src/sqlfluff/dialects/dialect_ansi.py

```diff
@@ -2804,6 +2816,7 @@ class SelectStatementSegment(BaseSegment):
     match_grammar = UnorderedSelectStatementSegment.match_grammar.copy(
         insert=[
             Ref("OrderByClauseSegment", optional=True),
+            Ref("OffsetClauseSegment", optional=True),
```

Comment:
According to https://jakewheat.github.io/sql-overview/sql-2011-foundation-grammar.html#order-by-clause, after `OrderBy`.